GiftModels/FinancialDetails.cs

```diff
@@ -32,7 +32,28 @@ public class FinancialDetails
 
         public virtual string FundType { get; set; }
 
-        public virtual string FisAccount { get; set; }
+        public virtual string Purpose { get; set; }
+
+        public virtual string FisAccount {

```

Comment:
@jpknoll When I look at the meta data for this in Giving, all I see is a get;set; Not this code. Is it actually being called?